mods/ra/sequences/aircraft.yaml

```diff
@@ -23,7 +23,7 @@ heli:
 	icon: heliicon
 
 hind:
```

Comment:
It would then need a lot more changes to the legacy map import and a map update rule.